### PR TITLE
feat : Added different types of buttons.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (332)

### CODE REVIEW

File: components/Buttons/Buttons.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.link-btn{
+    height: 2rem;
+    width: 4rem;
+    background-color: var(--primary-color);
+    border: 2px solid var(--white);
+    color: white;
+    font-size: large;
+    font-weight: 400;
```
Comment (stutisk) [inline components/Buttons/Buttons.css:28]: font weight can also be stored in variables 
height and width are repeated you can create a separate class for them

**Review by stutisk — COMMENTED** [review]
everything else looks good

File: components/Buttons/Buttons.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    font-size: var(--text-size-s);
+    border-radius: var(--border-radius-l);
+}
+.outlined-primary-btn{
+    padding: 0.25rem 0.5rem;
+    height: 2rem;
+    width: 5rem;
+    font-size: var(--text-size-s);
+    border-radius: var(--border-radius-l);
+    border: 2.5px solid var(--primary-color);
```
Comment (vidhi-thakur) [inline components/Buttons/Buttons.css:17]: Try to use rem in your code

File: components/Buttons/Buttons.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    border: 2px solid var(--white);
+    color: white;
+    font-size: large;
+    font-weight: 400;
+}
+
+.icon-btn{
+    padding: 0.25rem 0.5rem;
+    height: 2rem;
+    width: rem;
```
Comment (vidhi-thakur) [inline components/Buttons/Buttons.css]: I think you forget to write value before rem. Please update.

File: components/Cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,118 @@
+body{
+    background-color: var(--white);
+}
+.cards{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 20px 5px;
```
Comment (vidhi-thakur) [inline components/Cards/cards.css:7]: Use rem please

**Review by vidhi-thakur — APPROVED** [review]

**Comment by vidhi-thakur** [comment]
Added minor suggestions

**Pull request merged**